internal/wasm/store.go

```diff
@@ -196,11 +209,24 @@ type (
 		Buffer []byte
 		Min    uint32
 		Max    *uint32
+
+		// address is the memoryaddr(https://www.w3.org/TR/wasm-core-1/#syntax-tableaddr) of this memory instance.
+		// In other words, this equals the index of this memory instance in store.Memories.
+		address memoryAddress
 	}
 
 	// FunctionAddress is funcaddr (https://www.w3.org/TR/2019/REC-wasm-core-1-20191205/#syntax-funcaddr),
```

Comment:
Same comment vs above.. probably you can simplify docs by linking to the other.

Ex.

```
// XXIndex holds a copy of the last value of xx.index.
```

Reply:
actually these look like types, so just delete them